cdap-docs/included-applications/source/etl/plugins/transforms/script.rst

```diff
@@ -72,6 +76,15 @@ schema is the same as the input schema.
           {\"name\":\"tax\",\"type\":\"double\"},
           {\"name\":\"total\",\"type\":\"double\"}
         ]
+      }",
+      "lookup": "{
+        \"purchases\":{
+          \"type\":\"DATASET\",
+          \"properties\":{
+            \"dataset_argument1\":\"foo\",
+            \"dataset_argument2\":\"bar\"
```

Comment:
Do we have docs we can point to explaining how to do these arguments? (I know, I should know...)